packages/models-library/src/models_library/resource_tracker.py

```diff
@@ -85,3 +87,47 @@ class PricingAndHardwareInfoTuple(NamedTuple):
 class PricingPlanAndUnitIdsTuple(NamedTuple):
     pricing_plan_id: PricingPlanId
     pricing_unit_id: PricingUnitId
+
+
+# Filtering for listing service runs/usages
+
+
+class StartedAt(BaseModel):
+    from_: datetime | None = Field(None, alias="from")
+    until: datetime | None = Field(None)
```

Comment:
TIP: thse could also be more redable
```suggestion
    date_from: datetime | None = Field(None, alias="from")
    date_to: datetime | None = Field(None)
```